registry/tenets.yaml

```diff
@@ -155,8 +155,148 @@ codelingo:
       - kubernetes
       - golang
       - go
+  juju:
+    description: Best practices for the juju/juju repository
+    version: 0.0.0
+    tenets:
+      global-var:
+        hasIssues: false
+      exported-var-outside-export-test:
+        hasIssues: false
```

Comment:
All mocks should have an issue, which is that the CLQL query is not implemented.